base/applications/mspaint/lang/it-IT.rc

```diff
@@ -15,23 +16,23 @@ BEGIN
     POPUP "&File"
     BEGIN
         MENUITEM "Nuovo\tCtrl+N", IDM_FILENEW
-        MENUITEM "Apri...\tCtrl+F12", IDM_FILEOPEN
-        MENUITEM "Salva\tMaiusc+F12", IDM_FILESAVE
-        MENUITEM "Sal&va con nome...", IDM_FILESAVEAS
+        MENUITEM "&Apri...\tCtrl+F12", IDM_FILEOPEN
+        MENUITEM "&Salva\tMaiusc+F12", IDM_FILESAVE
+        MENUITEM "Sa&lva con nome...", IDM_FILESAVEAS
         MENUITEM SEPARATOR
-        MENUITEM "From Scanner or Camera...", IDM_FILEFROMSCANNERORCAMERA
+        MENUITEM "Da Scanner o fotocamera...", IDM_FILEFROMSCANNERORCAMERA
         MENUITEM SEPARATOR
-        MENUITEM "Page View", IDM_FILEPAGEVIEW
-        MENUITEM "Page Setup...", IDM_FILEPAGESETUP
-        MENUITEM "Print...\tCtrl+P", IDM_FILEPRINT
+        MENUITEM "Anteprima di stampa", IDM_FILEPAGEVIEW
```

Comment:
accel? My XPSP3 does have one for this string. So I do assume italian 2k3 has one too.